zookeeper-client/zookeeper-client-c/tests/TestClient.cc

```diff
@@ -207,7 +207,10 @@ class Zookeeper_simpleSystem : public CPPUNIT_NS::TestFixture
 #ifdef THREADED
     CPPUNIT_TEST(testNullData);
 #ifdef ZOO_IPV6_ENABLED
-    CPPUNIT_TEST(testIPV6);
+    //CPPUNIT_TEST(testIPV6);
```

Comment:
typo? we can't commit with the test turned off.

Reply:
Yeah. On my local setup, these tests were failing. I'm correcting this and pushing the change.